.github/workflows/build.yml

```diff
@@ -15,6 +15,8 @@ jobs:
       - uses: actions/checkout@v4
       - run: rustup toolchain install stable --profile minimal --no-self-update
       - uses: actions/setup-python@v5
+        with:
+            python-version: 3.10
```

Comment:
oh I see, this was using the `.python-version` file before -- maybe just revert the change to that that bumped it to 3.11?